File: bench/tasks/19-env-default/go/main.go
package main

import (
	"fmt"
	"os"
)

func main() {
	_ = os.Setenv("VOL_DENSITY_PORT", "9090")
	if v, ok := os.LookupEnv("VOL_DENSITY_PORT"); ok {
		fmt.Println(v)
	} else {
		fmt.Println("missing")
	}
	if v, ok := os.LookupEnv("VOL_DENSITY_NO_SUCH_KEY"); ok {
		fmt.Println(v)
	} else {
		fmt.Println("fallback")
	}
}

File: bench/tasks/19-env-default/zig/main.zig
const std = @import("std");

pub fn main() !void {
    const stdout = std.io.getStdOut().writer();
    try std.posix.setenv("VOL_DENSITY_PORT", "9090", true);

    if (std.posix.getenv("VOL_DENSITY_PORT")) |v| {
        try stdout.print("{s}\n", .{v});
    } else {
        try stdout.print("missing\n", .{});
    }
    if (std.posix.getenv("VOL_DENSITY_NO_SUCH_KEY")) |v| {
        try stdout.print("{s}\n", .{v});
    } else {
        try stdout.print("fallback\n", .{});
    }
}
